internal/wal_fetch_handler.go

```diff
@@ -128,6 +128,29 @@ func checkWALFileMagic(prefetched string) error {
 	return nil
 }
 
+// DownloadFileWithoutExtension downloads, decompresses and decrypts writes to stdout
+func DownloadFileWithoutDecompressor(backup *Backup, writeCloser io.WriteCloser) error {
+	defer writeCloser.Close()
+
+	for _, decompressor := range compression.Decompressors {
+		archiveReader, exists, err := TryDownloadFile(backup.BaseBackupFolder, backup.Name)
+		if err != nil {
+			return err
+		}
+		if !exists {
+			continue
+		}
+
+		err = DecompressDecryptBytes(&EmptyWriteIgnorer{WriteCloser: writeCloser}, archiveReader, decompressor)
```

Comment:
don't get this one, function called 'WithoutDecompressor' but we try to decompress, I'm really don't get it

Reply:
yes, I fixed it